Helper/Config.php

```diff
@@ -754,7 +754,7 @@ public function getTitle($storeId = null)
      */
     public function getApiKey($storeId = null)
     {
-        return $this->getEncryptedKey(self::XML_PATH_API_KEY, $storeId);
+        return $this->getEncryptedKey(self::XML_PATH_API_KEY, $storeId) ?: '';
```

Comment:
If API key is empty we can not proceed. Maybe check for empty and throw exception?

Reply:
This is helper function to return configuration value. I think only the functions that call it can decide whether to throw exception or not.

Reply:
@vitaliyreznikov  We faced similar issue with the function `explode` (Fixed in the [PR 1572](https://github.com/BoltApp/bolt-magento2/pull/1572)) so we would better fix it within the source rather than caller .